frontend/src/views/WheresMyGene/components/GeneSearchBar/components/SaveExport/index.tsx

```diff
@@ -227,113 +252,143 @@ export default function SaveExport({
         onClose={handleButtonClick}
         isCloseButtonShown={false}
       >
-        <div>
+        <StyledModalContent>
           <StyledSection>
             <StyledTitle>FIGURE</StyledTitle>
-            <StyledDiv>
-              <StyledFormControlLabel
-                control={
-                  <StyledInputCheckBox
-                    checked={selectedFileTypes.includes("png")}
-                  />
-                }
+            <StyledInputCheckboxWrapper>
+              <InputCheckbox
                 label="PNG"
+                caption="PNG CAPTION"
                 onChange={() => selectFileType("png")}
+                checked={selectedFileTypes.includes("png")}
                 data-testid="png-checkbox"
               />
-
-              <StyledFormControlLabel
-                control={
-                  <StyledInputCheckBox
-                    checked={selectedFileTypes.includes("svg")}
-                  />
-                }
+            </StyledInputCheckboxWrapper>
+            <StyledInputCheckboxWrapper>
+              <InputCheckbox
                 label="SVG"
+                caption="SVG CAPTION"
                 onChange={() => selectFileType("svg")}
+                checked={selectedFileTypes.includes("svg")}
                 data-testid="svg-checkbox"
               />
-            </StyledDiv>
+            </StyledInputCheckboxWrapper>
           </StyledSection>
+
           <StyledSection>
             <StyledTitle>DATA</StyledTitle>
-            <StyledDiv>
-              <StyledFormControlLabel
-                control={
-                  <StyledInputCheckBox
-                    checked={selectedFileTypes.includes("csv")}
-                  />
-                }
+            <StyledInputCheckboxWrapper>
+              <InputCheckbox
                 label="CSV"
+                caption="CSV CAPTION"
                 onChange={() => selectFileType("csv")}
+                checked={selectedFileTypes.includes("csv")}
                 data-testid="csv-checkbox"
               />
-            </StyledDiv>
+            </StyledInputCheckboxWrapper>
           </StyledSection>
-        </div>
-
-        <StyledButtonContainer>
-          <DownloadButton
-            sdsType="secondary"
-            sdsStyle="minimal"
-            onClick={handleButtonClick}
-          >
-            Cancel
-          </DownloadButton>
-          <DownloadButton
-            sdsStyle="square"
-            sdsType="primary"
-            sdsSize="large"
-            onClick={handleDownload}
-            disabled={!selectedFileTypes.length}
-            data-testid="dialog-download-button"
-          >
-            Download
-          </DownloadButton>
-        </StyledButtonContainer>
+
+          <UnderlyingDataChangeBanner width="100%" />
```

Comment:
The banner will appear above the download button in the modal. Setting width to 100% here to fill the width of the modal